sklearn/linear_model/passive_aggressive.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, C=1.0, fit_intercept=True,
                                    n_jobs=n_jobs)
         self.C = C
         self.loss = loss
+        self.alpha = 1.0
```

Comment:
I might have time next week to review it though @mblondel or @larsmans looking at it would probably be more valuable ;)